code/modules/research/techweb/all_nodes.dm

```diff
@@ -551,7 +551,7 @@
 	display_name = "Beam Weaponry"
 	description = "Various basic beam weapons"
 	prereq_ids = list("adv_weaponry")
-	design_ids = list("beamrifle", "ioncarbine")
+	design_ids = list("temp_gun", "xray_laser")

```

Comment:
>xray
>basic

Reply:
xray now is really bad and never used gameplay wise

Reply:
doesn't xray still annihilate blobs

Reply:
they stop after a blob hit now iirc

Reply:
fair fair

Reply:
@ShadowDeath6 they are just lasers that give you rads and shoot through walls now.